pkg/registry/serving/composer.go

```diff
@@ -106,15 +111,14 @@ func Compose(sdep *seldonv1.SeldonDeployment) error {
 				}
 			}
 
+			// Compose SeldonPodSpec
+			if err := composeSeldonPodSpec(&p.Graph, componentSpecMap); err != nil {
```

Comment:
Can we use p here?

Reply:
No， because composeSeldonPodSpec will call composeSeldonPodSpec  recursively to set every graph node. as follow:
https://github.com/kleveross/klever-model-registry/blob/master/pkg/registry/serving/composer.go#L301